providers/fab/src/airflow/providers/fab/auth_manager/fab_auth_manager.py

```diff
@@ -340,14 +342,30 @@ def is_authorized_dag(
             method=dag_method, details=details, user=user
         ):
             return False
-
+        
+        # return all(
+        #     (
+        #         self._is_authorized(method=method, resource_type=resource_type, user=user)
+        #         if resource_type != RESOURCE_DAG_RUN or not hasattr(permissions, "resource_name")
+        #         else self._is_authorized_dag_run(method=method, details=details, user=user)
+        #     )
+        #     for resource_type in resource_types
+        # )
+
+        # if Airflow version is less than 3.1.0 and the resource type is RESOURCE_HITL_DETAIL, skip.
         return all(
             (
-                self._is_authorized(method=method, resource_type=resource_type, user=user)
-                if resource_type != RESOURCE_DAG_RUN or not hasattr(permissions, "resource_name")
-                else self._is_authorized_dag_run(method=method, details=details, user=user)
-            )
-            for resource_type in resource_types
+                True
+                if (
+                    resource_type == RESOURCE_HITL_DETAIL
+                    and packaging.version.parse(packaging.version.parse(airflow_version).base_version) < packaging.version.parse("3.1.0")
```

Comment:
```suggestion
                    and AIRFLOW_V_3_1_PLUS
```

from airflow.providers.fab.version_compat import AIRFLOW_V_3_1_PLUS

Reply:
Oh yeah we have this in version compat, will make the update

Reply:
looks like `AIRFLOW_V_3_1_PLUS` will be true if Airflow version is greater than or equal to 3.1.0. If we want to make it 3.1.0 only, then this might not work?

Reply:
Yes, but I think we probably would like to make it higher than 3.1.0 ?

Reply:
Looks like it's outdated. I'll mark it as resolved. But feel free to reopen it if there's more to discuss 🙂